### DISCUSSION TITLE
onlyfans drm

### DISCUSSION THREAD

**Discussion by dbzgtaf22** [aclap-dev/video-downloadhelper#591] [Help and Q&A]
### Your question

onlyfans drm videos not downloading

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://onlyfans.com/my/collections/bookmarks/all",
  "error": {
    "details": "Error: Native port disconnected",
    "downloadable_id": "downloadable_1993421882442173",
    "error": "coapp_failure",
    "id": "coapp_failure_081d34c3-502f-4ebe-b5f3-aa2115fc138a"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://onlyfans.com/my/collections/bookmarks/all",
  "error": {
    "details": "Error: Native port disconnected",
    "downloadable_id": "downloadable_1993421882442173",
    "error": "coapp_failure",
    "id": "coapp_failure_081d34c3-502f-4ebe-b5f3-aa2115fc138a"
  }
}_

**Comment by paulrouget** [comment]
We do not support DRM: https://github.com/aclap-dev/video-downloadhelper/wiki/What-can-I-do-if-the-video-is-protected-by-DRM%3F

**Reply by GrampaWildWilly** [reply]
See #592 for an example of how you can verify the presence of DRM protection for yourself.
